tsconfig.json

```diff
@@ -4,7 +4,7 @@
 		"target": "ES2020",
 		"module": "commonjs",
 		"allowJs": false,
-		"sourceMap": true,
```

Comment:
It's not needed, since during development we should be using `ts-node` (via `npm start` or `npm run cli`), which always shows source locations. Though it can't hurt, so I'll add it back in.

Reply:
Good point. I don't mind it being included or not since it doesn't matter, so up to you.